css/popup-window.css

```diff
@@ -0,0 +1,224 @@
+.modal {
+  position: fixed;
+  z-index: 1;
+  background-color: rgb(255, 255, 255, 0.9);
+  backdrop-filter: blur(9px);
+  overflow-y: hidden;
+  overflow-x: hidden;
+  width: 80%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 24px;
+  left: 10%;
+  top: -5%;
+}
+
+.frame-desktop {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.close-popup {
+  font-size: 30px;
+  color: #67798e;
+}
+
+.close-popup:hover,
+.close-popup:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.img-popup {
+  display: none;
+}
+
+.img-popup-mobile {
+  width: 100%;
+  padding-top: 20px;
+}
+
+.paragraph-popup {
+  display: none;
+}
+
+.paragraph-popup-mobile {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.client {
+  font-style: normal;
+  font-weight: 400;
+  text-transform: uppercase;
+  color: #344563;
+  margin-left: 4px;
+}
+
+.role {
+  font-style: normal;
+  font-weight: 400;
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+}
+
+.year {
+  font-style: normal;
+  font-weight: 400;
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+}
+
+.tags-hidden {
+  list-style: none;
+  flex-direction: row;
+  margin-left: 6px;
+  display: none;
+  text-decoration: none;
+}
+
+.tags {
+  padding-bottom: 20px;
+  list-style: none;
+  text-decoration: none;
+}
+
+.btn-block {
+  display: flex;
+  justify-content: space-between;
+}
+
+.btn-popup-right {
+  width: 145px;
+  height: 48px;
+}
+
+.btn-popup-left {
+  width: 145px;
+  height: 48px;
+}
```

Comment:
- [ ] Please 🙏🏽 use the styles for the active and hover button to be similar to the Figma template 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/232170760-4cf40646-441b-484e-aed3-716b24601ac9.png)